src/CommandPool.php

```diff
@@ -47,9 +49,12 @@ public function __construct(
         if (!isset($config['concurrency'])) {
             $config['concurrency'] = 25;
         }
+        if (!array_key_exists('preserve_iterator_keys', $config)) {
```

Comment:
Instead of setting the config value here, the conditional below should just be:
```php
if (!empty($config['preserve_iterator_keys']))
```

Reply:
Sure. I'll use `if (empty())` though. :-D

Reply:
>.> Yep!